src/identifier.js

```diff
@@ -17,6 +17,8 @@ export const keywords = {
   6: ecma5AndLessKeywords + " const class extends export import super"
 }
 
+export const keywordRelationalOperators = [ "in", "instanceof" ]
```

Comment:
Why not a regexp in line with our other checks?

Reply:
I updated the commit.